### DISCUSSION TITLE
Consider restricting the plugin to particular files via options.include

### DISCUSSION THREAD

**Discussion by hubeishuaige** [rollup/plugins#1586] [Q&A]
use rollup-plugin-inject ，tip： rollup-plugin-inject: failed to parse  xxxxx   Consider restricting the plugin to particular files via options.include

![image](https://github.com/rollup/plugins/assets/21007762/3da93192-632e-4140-8735-972c49ea8f0f)
Although it does not affect my project permission, I am still very unhappy to see it

**Comment by jordanade** [comment]
Same problem here. It seems to be triggered by an sass import from node_modules in my case:
```@import "~bootstrap-sass/assets/stylesheets/bootstrap";```
